tests/c/bfbug.c

```diff
@@ -0,0 +1,165 @@
+// Run-time:
+//   env-var: YKD_SERIALISE_COMPILATION=0
+//   env-var: YKD_PRINT_JITSTATE=1
+//   stderr:
+//     ...
+//     jit-state: enter-jit-code
+//     jit-state: exit-jit-code
+//     ...
+//     jit-state: enter-stopgap
+//     ...
+//   stdin:
+//     1234
+
+// This is bf_base.c from https://github.com/ykjit/ykcbf modified to hard-code the input to the
+// interpreter (hello.bf from the same repo).
+
+#include <err.h>
+#include <fcntl.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/stat.h>
+#include <unistd.h>
+#include <yk.h>
+#include <yk_testing.h>
+
+#define CELLS_LEN 30000
+#define INPUT_PROG                                                             \
```

Comment:
We can't remove the copyright on this: we need to credit Daniel B Cristofani.

Reply:
Fixed in [830647f](https://github.com/ykjit/yk/pull/552/commits/830647fbefcdbf3989cdfb2bed7b5311acdd18ba)